lib/Util.ts

```diff
@@ -1,6 +1,15 @@
 import axios, { AxiosRequestConfig, AxiosResponse } from "axios"
 import { REST_URL } from "./BITBOX"
 
+// Pull in additional libraries required to support the sweep function.
+import { Address } from "./Address"
+const addressLib = new Address(REST_URL)
+import { ECPair } from "./ECPair"
+const ecPair = new ECPair(addressLib)
+import { TransactionBuilder } from "./TransactionBuilder"
+import { BitcoinCash } from "./BitcoinCash"
+const bitcoinCash = new BitcoinCash(addressLib)
+
 export interface AddressDetails {
```

Comment:
These are all libraries that need to be pulled in to perform the sweep functionality. Something about how I pulled these in are causing the unit tests to fail. Even if they pass, all these cross-linked libraries are a red flag, which makes me think this sweep functionality should go into a library higher up in the stack. e.g. BITBOX contains primitives that sweep makes use of. But sweeping is not a primitive, so it probably doesn't belong in BITBOX.

Reply:
all the `imports` should be on the top;

the `consts` should be below the `imports`;

Reply:
also try using 
`import Address from "./Address"` syntax if you are getting errors.

versus

`import { Address } from "./Address"`